src/feature/organization/organizationMain/components/section/OrganizationMainButtonsSection.tsx

```diff
@@ -0,0 +1,35 @@
+import styled from "@emotion/styled";
+import ManagementButtons from "feature/organization/organizationMain/components/buttons/ManagementButtons";
+import MemberButtons from "feature/organization/organizationMain/components/buttons/MemberButtons";
+import VisitorButtons from "feature/organization/organizationMain/components/buttons/VisitorButtons";
+import ActionButtons from "feature/organization/organizationMain/components/buttons/ActionButtons";
+
+interface Props {
+  organizationId: string | number | string[];
+  userAuth: number;
+}
+
+const OrganizationMainButtonsSection: React.FC<Props> = ({ userAuth, organizationId }) => {
+  let AuthButtonsComponent;
+
+  if (userAuth === 0) {
+    AuthButtonsComponent = ManagementButtons;
+  } else if (userAuth === 1) {
+    AuthButtonsComponent = MemberButtons;
+  } else {
+    AuthButtonsComponent = VisitorButtons;
+  }
```

Comment:
`if ~ else if ~ else` 가 중첩되어 가독성이 약간 좋지 않은 것 같습니다. 또한 let 사용으로 인해 불변성 보장을 받지 못할 것 같은데 혹시 아래와 같이 또한 컴포넌트 외부에서 처리 및 리렌더 될 때마다 새로 불러오지 않으므로 이 부분만 컴포넌트 위로 빼는 것은 어떨까요? ( 이렇게 하면 `const`  사용으로도 변경할 수 있을 것 같습니다!)

아래와 같은 형식을 생각하였습니다. 

```suggestion

const AUTH_BUTTONS_COMPONENT = {
  0: ManagementButtons, 
  1: MemberButtons, 
  2: VisitorButtons, 
}

  const AuthButtonComponent = AUTH_BUTTONS_COMPONENT[userAuth]
  
  
  return (
  ...
    <AuthButtonComponent /> 
  )
```

Reply:
오.. 넵 좋습니다.
제안해주신 사항 반영해서 수정했습니다!